src/cli/standard_compiler.rs

```diff
@@ -0,0 +1,68 @@
+use std::process::Command;
```

Comment:
(i'm commenting on the first line but it applies on the whole file)

I moved every compiler in `crate::compilers::*`. `StandardCompiler` can be found there.